docs/snippets/angular/list-story-template.ts.mdx

```diff
@@ -0,0 +1,31 @@
+```ts
+// List.stories.ts
+
+import { Story } from '@storybook/angular/types-6-0';
+
+import ListComponent from './List.component';
+import ListItemComponent from './ListItem.component';
+
+import { Unchecked } from './ListItem.stories';
+
+const ListTemplate: Story<ListComponent> = (args: ListComponent) => ({
```

Comment:
no export default on this stroy?

Reply:
@ThibaudAV  `ListTemplate` is a template in this case, not a story.